src/main/java/frc/team2412/robot/commands/autonomous/TwoBallFenderAutoCommand.java

```diff
@@ -0,0 +1,49 @@
+package frc.team2412.robot.commands.autonomous;
+
+import java.util.List;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.subsystem.DrivebaseSubsystem;
+
+public class TwoBallFenderAutoCommand extends SequentialCommandGroup {
+    private final DrivebaseSubsystem drivebaseSubsystem;
+
+    public TwoBallFenderAutoCommand(DrivebaseSubsystem drivebaseSubsystem) {
+        this.drivebaseSubsystem = drivebaseSubsystem;
+
+        TrajectoryConfig fastConfig = new TrajectoryConfig(1, 0.8);
+        TrajectoryConfig slowConfig = new TrajectoryConfig(0.2, 1);
+
+        Trajectory trajectory1 = TrajectoryGenerator.generateTrajectory(
```

Comment:
It would be great to include comments to describe the starting pose of the robot (for testing and drive team), what this path does and why we might want to use it can be quickly understood, setup and used under the pressure of an impending match.

Reply:
I'm now including images of the field and where to set robot up on it.

Reply:
In the past we've had the autonomous chooser display an image of the field and robot with the path it would follow after drive team selected an autonomous mode. Not sure if it would be worth adding in now, though (haven't been able to find an easy way to do this in Shuffleboard).